src/components/ChatLog.jsx

```diff
@@ -0,0 +1,34 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+function ChatLog({ entries, onLikeToggle }) {
+  return (
+    <section className="chat-log">
+      {entries.map((entry) => (
+        <ChatEntry
+          key={entry.id}
+          {...entry}
+          onLikeToggle={() => onLikeToggle(entry.id)}
+          isLocal={entry.sender === 'Vladimir'}
+        />
+      ))}
+    </section>
+  );
+}
```

Comment:
It's more typical to see the result of using `map` referenced by a variable and using that variable in the JSX, like:
```js
const chatEntries = entries.map((entry) => (
        <ChatEntry
          key={entry.id}
          {...entry}
          onLikeToggle={() => onLikeToggle(entry.id)}
          isLocal={entry.sender === 'Vladimir'}
        />
      ));

return (
  <section className="chat-log">
    {chatEntries}
  </section>
);
```